[2025-03-19T08:12:14.967+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.transform scheduled__2025-03-19T07:10:00+00:00 [queued]>
[2025-03-19T08:12:14.973+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.transform scheduled__2025-03-19T07:10:00+00:00 [queued]>
[2025-03-19T08:12:14.973+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-03-19T08:12:14.981+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): transform> on 2025-03-19 07:10:00+00:00
[2025-03-19T08:12:14.985+0000] {standard_task_runner.py:57} INFO - Started process 331 to run task
[2025-03-19T08:12:14.987+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'etl_pipeline', 'transform', 'scheduled__2025-03-19T07:10:00+00:00', '--job-id', '81', '--raw', '--subdir', 'DAGS_FOLDER/etl_dags.py', '--cfg-path', '/tmp/tmpjyk5roqx']
[2025-03-19T08:12:14.988+0000] {standard_task_runner.py:85} INFO - Job 81: Subtask transform
[2025-03-19T08:12:14.999+0000] {logging_mixin.py:154} WARNING - /home/airflow/.local/lib/python3.8/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-19T08:12:15.021+0000] {task_command.py:416} INFO - Running <TaskInstance: etl_pipeline.transform scheduled__2025-03-19T07:10:00+00:00 [running]> on host 9431682cb7a1
[2025-03-19T08:12:15.067+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2025-03-19T07:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-19T07:10:00+00:00'
[2025-03-19T08:12:15.106+0000] {logging_mixin.py:154} INFO - Index(['location_id', 'aqi_us', 'main_us', 'aqi_cn', 'main_cn', 'timestamp',
       'aqi_us_category', 'aqi_cn_category', 'aqi_avg', 'main_pollutant',
       'severe_pollution', 'traffic_id', 'weather_id', 'fact_id'],
      dtype='object')
[2025-03-19T08:12:15.112+0000] {logging_mixin.py:154} INFO - ✅ Dữ liệu đã được xử lý và lưu vào processed_data
[2025-03-19T08:12:15.113+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-03-19T08:12:15.119+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=transform, execution_date=20250319T071000, start_date=20250319T081214, end_date=20250319T081215
[2025-03-19T08:12:15.161+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-19T08:12:15.174+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
